### PR TITLE
feat(stack-packs): adds links to some wordpress stack strings

### PR DESCRIPTION
**Pull request by housseindjirdeh**
**Summary**

Adds links for search results + more to Wordpress stack pack audits

@wardpeet @paulirish 

**Related Issues/PRs**

#7243

### CODE REVIEW

File: stack-packs/packs/wordpress.js [hunk 1]
```diff
@@ -17,28 +17,28 @@ module.exports = {
...
+      'A number of [WordPress plugins](https://wordpress.org/plugins/search/minify+css/) can speed up your site by concatenating, minifying, and compressing your styles. You may also want to use a build process to do this minification up-front if possible.',
     'unminified-javascript':
-      'A number of WordPress plugins can speed up your site by concatenating, minifying, and compressing your scripts. You may also want to use a build process to do this minification up-front if possible',
+      'A number of [WordPress plugins](https://wordpress.org/plugins/search/minify+javascript/) can speed up your site by concatenating, minifying, and compressing your scripts. You may also want to use a build process to do this minification up-front if possible',
     'efficient-animated-content':
-      'Consider uploading your GIF to a service which will then make it available to embed an HTML5 video. Alternatively find a plugin that will automatically convert GIF images into videos upon upload.',
+      'Consider uploading your GIF to a service which will then make it available to embed an HTML5 video.',
```
Comment (housseindjirdeh) [inline stack-packs/packs/wordpress.js]: Could not find a single plugin that automatically converts GIFs to videos, so removed this mention.

File: stack-packs/packs/wordpress.js [hunk 2]
```diff
@@ -17,28 +17,28 @@ module.exports = {
...
     'uses-long-cache-ttl':
-      'Read more about [Browser Caching in WordPress](https://codex.wordpress.org/WordPress_Optimization#Browser_Caching).',
+      'Read about [Browser Caching in WordPress](https://codex.wordpress.org/WordPress_Optimization#Browser_Caching).',
     'uses-optimized-images':
-      'Consider using a WordPress plugin that optimizes your images, automatically serves the required sizes, and serves them from a CDN.',
+      'Consider using a [WordPress plugin](https://wordpress.org/plugins/search/optimize+images/) that optimizes your images, automatically serves the required sizes, and serves them from a CDN.',
```
Comment (paulirish) [inline stack-packs/packs/wordpress.js]: > automatically serves the required sizes

this is the uses-responses-images audit

> and serves them from a CDN.

Lighthouse doesn't have strong feeling about the images being served from a CDN, per say. I would remove this bit.

Comment (paulirish) [inline stack-packs/packs/wordpress.js]:
```suggestion
      'Consider using an [image optimization WordPress plugin](https://wordpress.org/plugins/search/optimize+images/) that compresses your images while retaining quality.',
```

just a fan of link text being descriptive ;)

File: stack-packs/packs/wordpress.js [hunk 3]
```diff
@@ -17,28 +17,28 @@ module.exports = {
...
     'render-blocking-resources':
-      'There a number of WordPress plugins that can help you inline critical assets or defer less important resources. Beware that optimizations provided by these plugins may break features of your theme or plugins, so you will likely need to make code changes.',
+      'There a number of WordPress plugins that can help you [inline critical assets](https://wordpress.org/plugins/search/critical+css/) or [defer less important resources](https://wordpress.org/plugins/search/defer+css+javascript/). Beware that optimizations provided by these plugins may break features of your theme or plugins, so you will likely need to make code changes.',
     'unminified-css':
-      'A number of WordPress plugins can speed up your site by concatenating, minifying, and compressing your styles. You may also want to use a build process to do this minification up-front if possible.',
+      'A number of [WordPress plugins](https://wordpress.org/plugins/search/minify+css/) can speed up your site by concatenating, minifying, and compressing your styles. You may also want to use a build process to do this minification up-front if possible.',
```
Comment (paulirish) [inline stack-packs/packs/wordpress.js]: > You may also want to use a build process to do this minification up-front if possible

Is this possible in WP world? In my experience all CSS/JS comes from either the theme or a plugin, and you can't change those assets with a build process.

**Comment by paulirish** [comment]
Shane had some comments over in https://github.com/GoogleChrome/lighthouse/pull/7243#pullrequestreview-223948160 that apply here.

File: stack-packs/packs/wordpress.js [hunk 4]
```diff
@@ -17,28 +17,28 @@ module.exports = {
...
     'unminified-javascript':
-      'A number of WordPress plugins can speed up your site by concatenating, minifying, and compressing your scripts. You may also want to use a build process to do this minification up-front if possible',
+      'A number of [WordPress plugins](https://wordpress.org/plugins/search/minify+javascript/) can speed up your site by concatenating, minifying, and compressing your scripts. You may also want to use a build process to do this minification up-front if possible',
```
Comment (exterkamp) [inline stack-packs/packs/wordpress.js]:
```suggestion
      'A number of [WordPress plugins](https://wordpress.org/plugins/search/minify+javascript/) can speed up your site by concatenating, minifying, and compressing your scripts. You may also want to use a build process to do this minification up-front if possible.',
```

File: stack-packs/packs/wordpress.js (hunk 1 above)
Comment (exterkamp) [inline stack-packs/packs/wordpress.js]:
```suggestion
      'Consider uploading your GIF to a service which will make it available to embed as an HTML5 video.',
```

File: stack-packs/packs/wordpress.js [hunk 5]
```diff
@@ -17,28 +17,28 @@ module.exports = {
...
     'uses-text-compression': 'You can enable this in your web server configuration.',
     'uses-responsive-images':
-      'Upload images directly through the media library to ensure that the required image sizes are available, and then insert them from the media library or use the image widget to ensure the optimal image sizes are used (including those for the responsive breakpoints). Avoid using `Full Size` images unless the dimensions are adequate for their usage. [Learn More](https://codex.wordpress.org/Inserting_Images_into_Posts_and_Pages#Image_Size)',
+      'Upload images directly through the [media library](https://codex.wordpress.org/Media_Library_Screen) to ensure that the required image sizes are available, and then insert them from the media library or use the image widget to ensure the optimal image sizes are used (including those for the responsive breakpoints). Avoid using `Full Size` images unless the dimensions are adequate for their usage. [Learn More](https://codex.wordpress.org/Inserting_Images_into_Posts_and_Pages#Image_Size)',
```
Comment (brendankenny) [inline stack-packs/packs/wordpress.js]:
```suggestion
      'Upload images directly through the [media library](https://codex.wordpress.org/Media_Library_Screen) to ensure that the required image sizes are available, and then insert them from the media library or use the image widget to ensure the optimal image sizes are used (including those for the responsive breakpoints). Avoid using `Full Size` images unless the dimensions are adequate for their usage. [Learn More](https://codex.wordpress.org/Inserting_Images_into_Posts_and_Pages#Image_Size).',
```

File: stack-packs/packs/wordpress.js (hunk 4 above)
Comment (brendankenny) [reply stack-packs/packs/wordpress.js]: also no hyphen for `up front`

**Pull request merged**